var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -416,11 +427,19 @@ def edit(self, spec, prefix):
             elpa_suffix = '_openmp' if '+openmp' in elpa else ''
             elpa_incdir = elpa.headers.directories[0]
 
-            fcflags += ['-I{0}'.format(os.path.join(elpa_incdir, 'modules'))]
-            libs.append(os.path.join(elpa.libs.directories[0],
-                                     ('libelpa{elpa_suffix}.{dso_suffix}'
-                                      .format(elpa_suffix=elpa_suffix,
-                                              dso_suffix=dso_suffix))))
+            fcflags += ['-I{0}'.format(join_path(elpa_incdir, 'modules'))]
+
+            # As a best practice used by CP2K developers,
+            # AOCC considering using static libraries of ELPA
```

Comment:
Then please drop it. The only [reason why CP2K is linking to ELPA statically](https://github.com/cp2k/cp2k/commit/6cf6e4d1c367a5a1f5f27c4844edd8583f652c9f#diff-8de432b66f5b5350591f4adab306dc691ededc81d56268b2a5786094668fb313) in the toolchain scripts was the difficulty of ensuring `-fPIC` and managing the different sets of OpenMP-enabled/-disabled libraries. Both should not be an issue with Spack.

Reply:
Currently AOCC supports static libraries of ELPA and support for shared library is ongoing.